src/App.tsx

```diff
@@ -1,14 +1,44 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
+import { getTodos } from './api';
+import { filterTodos } from './components/Helpers/FilterTodos';
+import { getErrorMessage } from './components/Helpers/GetErrorMessage';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [sortBy, setFilterBy] = useState('all');
```

Comment:
we are filtering, not sorting, filterType is a better name
also, create enum for storing possible values, don't forget to use it everywhere, not only here